docs/notifications.json

```
@@ -13,5 +13,10 @@
     "id": 78,
     "title": "<b>MUI X v6.18.x and the latest improvements before the next major</b>",
     "text": "New stable components, polished features, better performance and more. Check out the details in our <a style=\"color: inherit;\" data-ga-event-category=\"Announcement\" data-ga-event-action=\"notification\" data-ga-event-label=\"mui-x-end-v6\" href=\"https://mui.com/blog/mui-x-end-v6-features/\">recent blog post</a>."
+  },
+  {
+    "id": 79,
+    "title": "<b>The MUI Developer Survey 2023 is live</b>",
```

```suggestion
    "title": "<b>The 2023 MUI developer survey is live</b>",
```

Consistency with that other PR!

---

would this work?

```suggestion
    "title": "<b>A new Developer Survey is open</b>",
```

---

I'd make the title a call to action instead of just an informative statement, and the year is not confusing if we go with 2024! 😃 About dropping the "MUI" prefix here, though, we'd need to do this for all of the other editions (which aren't necessarily focused on just one of the products we develop). In that sense, maybe something like:

> **Participate in the 2024 Developer Survey**
> Influence our roadmap for 2024! Take a few minutes..."

---

```suggestion
    "title": "<b>Influence 2024's roadmap</b>",
```